util/mkdef.pl

```diff
@@ -40,6 +41,7 @@
            'ordinals=s' => \$ordinals_file,
            'version=s'  => \$version,
            'OS=s'       => \$OS,
+           'type'       => \$type,
```

Comment:
This isn't quite right, and is the reason for the build failures.

```suggestion
           'type=s'     => \$type,
```